plugins/hbase_logs.yaml

```diff
@@ -0,0 +1,132 @@
+version: 0.0.1
+title: Apache HBase
+description: Log parser for Apache HBase
+parameters:
+  - name: enable_master_log
+    type: bool
+    default: true
+  - name: master_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-master-*.log"
+  - name: enable_region_log
+    type: bool
+    default: true
+  - name: region_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-regionserver-*.log"
+  - name: enable_zookeeper_log
+    type: bool
+    default: false
+  - name: zookeeper_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-zookeeper-*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_master_log }}
+    filelog/master:
+      include:
+        {{ range $fp := .master_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+          line_start_pattern: '^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}\s+|^\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}'
+      start_at: {{ .start_at }}
+      operators:
+        - type: regex_parser
+          if:  '$record matches "^\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2},\\d{3}\\s+"'
+          regex: '^(?P<timestamp>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}),\d{3}\s+(?P<hbase_severity>[A-Z]*)\s*?\[(?P<thread>[^\]]*)\]\s(?P<hbase_source>[^:]*):\s(?P<message>(?s).*)\n'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%Y-%m-%d %H:%M:%S'
+          severity:
+            parse_from: attributes.hbase_severity
+            mapping:
+              warning: warn
+              emergency: fatal
```

Comment:
in OTEL, `warning` and `emergency` aren't valid aliases; instead, for these ones we'd use `warn` and `fatal`, which means we can actually omit this `mapping` section entirely

see: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/types/severity.md#how-to-simplify-configuration-with-a-preset (the default preset is `default`, so just omitting the mapping section is OK)